lib/paginate_firestore.dart

```diff
@@ -252,6 +256,54 @@ class _PaginateFirestoreState extends State<PaginateFirestore> {
 
     return listView;
   }
+
+  Widget _buildPageView(PaginationLoaded loadedState) {
+    var pageView = CustomScrollView(
+      reverse: widget.reverse,
+      shrinkWrap: widget.shrinkWrap,
+      controller: widget.scrollController,
+      scrollDirection: widget.scrollDirection,
+      physics: widget.physics,
+      slivers: <Widget>[
+        if (widget.header != null) SliverToBoxAdapter(child: widget.header),
```

Comment:
Same here. Sliver can be removed

Reply:
Agree

Reply:
Sorry, @jslattery26. I think there was a miscommunication here. I meant we can directly add the header or footer instead of wrapping it with Sliver. Like this

```dart
if (widget.header != null) widget.header,
```

Instead of this
```dart
if (widget.header != null) SliverToBoxAdapter(child: widget.header),
```